deployment/terraform/strings.go

```diff
@@ -319,6 +322,147 @@ Group=ubuntu
+          value: {{.ClusterName}}
+      metrics:
+        - name: CPUUtilization
+          statistics: [Average]
+        - name: DatabaseConnections
+          statistics: [Sum]
+        - name: FreeableMemory
+          statistics: [Average]
+        - name: FreeStorageSpace
+          statistics: [Average]
+        - name: ReadThroughput
+          statistics: [Average]
+        - name: WriteThroughput
+          statistics: [Average]
+        - name: ReadLatency
+          statistics: [Maximum]
+        - name: WriteLatency
+          statistics: [Maximum]
+        - name: ReadIOPS
+          statistics: [Average]
+        - name: WriteIOPS
+          statistics: [Average]
+    - type: AWS/ES
+      regions:
+        - us-east-1
+      period: {{.Period}}
+      length: {{.Length}}
+      delay: {{.Delay}}
+      addCloudwatchTimestamp: true
+      searchTags:
+        - key: ClusterName
+          value: {{.ClusterName}}
+      metrics:
+        - name: CPUUtilization
+          statistics: [Average]
+        - name: FreeStorageSpace
+          statistics: [Sum]
+        - name: ClusterStatus.green
+          statistics: [Maximum]
+        - name: ClusterStatus.yellow
+          statistics: [Maximum]
+        - name: ClusterStatus.red
+          statistics: [Maximum]
+        - name: Shards.active
+          statistics: [Sum]
+        - name: Shards.unassigned
+          statistics: [Sum]
+        - name: Shards.delayedUnassigned
+          statistics: [Sum]
+        - name: Shards.activePrimary
+          statistics: [Sum]
+        - name: Shards.initializing
+          statistics: [Sum]
+        - name: Shards.initializing
+          statistics: [Sum]
+        - name: Shards.relocating
+          statistics: [Sum]
+        - name: Nodes
+          statistics: [Maximum]
+        - name: SearchableDocuments
+          statistics: [Maximum]
+        - name: DeletedDocuments
+          statistics: [Maximum]
+    - type: AWS/EC2
+      regions:
+        - us-east-1
+      period: {{.Period}}
+      length: {{.Length}}
+      delay: {{.Delay}}
+      addCloudwatchTimestamp: true
+      searchTags:
+        - key: ClusterName
+          value: {{.ClusterName}}
+      metrics:
+        - name: CPUUtilization
+          statistics: [Average]
+        - name: NetworkIn
+          statistics: [Average, Sum]
+        - name: NetworkOut
+          statistics: [Average, Sum]
+        - name: NetworkPacketsIn
+          statistics: [Sum]
+        - name: NetworkPacketsOut
+          statistics: [Sum]
+        - name: DiskReadBytes
+          statistics: [Sum]
+        - name: DiskWriteBytes
+          statistics: [Sum]
+        - name: DiskReadOps
+          statistics: [Sum]
+        - name: DiskWriteOps
+          statistics: [Sum]
+        - name: StatusCheckFailed
+          statistics: [Sum]
+        - name: StatusCheckFailed_Instance
+          statistics: [Sum]
+        - name: StatusCheckFailed_System
+          statistics: [Sum]
```

Comment:
I don't think we need the EC2 metrics. 0/5

Reply:
Yeah, I added those when I was testing the whole thing and did not remove them even though I'm not using them in any of the panels. The only interesting ones may be the `StatusCheckFailed` ones to monitor if an instance is down. Let me see if I can add a panel with that info and I can remove the other ones.